### PR TITLE
#36 update sphere-node-sdk to the newest version

### PR DESCRIPTION
**Pull request by lojzatran**
Fixes: https://github.com/sphereio/sphere-stock-import/issues/36

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -46,7 +46,7 @@
     "csv": "0.3.7",
     "debug": "2.2.0",
     "optimist": "0.6.1",
-    "sphere-node-sdk": "1.7.0",
+    "sphere-node-sdk": "1.14.0",
```
Comment (butenkor) [inline package.json]: @lojzatran Please change it to: ^1.14.0 so that it matches in the future all feature versions automatically

Comment (lojzatran) [inline package.json]: https://github.com/sphereio/sphere-stock-import/pull/37/commits/711f71869c01552b90e821188a808843cd557054

**Comment by butenkor** [comment]
@emmenko @PhilippSpo  could you please review and create release?

File: package.json (hunk 1 above)
Comment (butenkor) [inline package.json]: Thanks

**Comment by adnasa** [comment]
@lojzatran I don't think [starting a commit message with `#` is a great idea](http://stackoverflow.com/questions/2788092/start-a-git-commit-message-with-a-hashmark)

**Comment by lojzatran** [comment]
@adnasa thanks, I did not know. But is it really a problem? I never had a problem with commit messages starting with #. I think it's only a problem when you're writing a commit message on your local machine, right? Does it affect Git server in any way?

**Pull request merged**